src/v/cluster/tests/cluster_test_fixture.h

```diff
@@ -95,7 +95,6 @@ class cluster_test_fixture {
 
     cluster_test_fixture()
```

Comment:
How does this work? Can more than one of these objects be created in tests (I assume yes)? Currently it creates/destroys the groups on construction, destruction, right?